src/components/App.js

```diff
@@ -1,44 +1,38 @@
 import React, {Component, Fragment} from 'react';
-import {Route} from 'react-router-dom';
+import {Route, Redirect} from 'react-router-dom';
+
 import {connect} from 'react-redux';
 import {fetchProducts} from '../redux-store/actions/product-actions';
-import {fetchUsers} from '../redux-store/actions/user-actions';
 
-import Home from './Home';
-import Cart from './Cart';
 import Navbar from './Nav';
 import Catalog from './Catalog';
+import Cart from './Cart';
 import SingleProduct from './SingleProduct';
 
-import '../styles/App.css';
-
 class App extends Component {
-  async componentDidMount() {
-    const {fetchProducts, fetchUsers} = this.props;
-    await Promise.all([fetchProducts(), fetchUsers()]);
+  componentDidMount() {
+    this.props.fetchProducts();
   }
-  render() {
-    const {products} = this.props;
 
+  render() {
     return (
       <div className="column-container">
-        <Navbar />
-        {/* Home route */}
-        <Route
-          exact
-          path="/"
-          render={props => <Home props={props} products={products} />}
-        />
+        <Route component={Navbar} />
+
+        {/* Landing page route */}
+        <Route exact path="/" render={() => <Redirect to="/catalog" />} />
```

Comment:
Yeah, sorry, I put the Redirect as filler because our Home component wasn't complete. I have `/catalog`, `/catalog/:category`, and `/catalog/search/:searchTerm` all render `Catalog`.